manifests/init.pp

```diff
@@ -79,10 +95,18 @@
     create_resources('facter::fact',$facts_hash_real, $facts_defaults)
   }
 
-  exec { "mkdir_p-${facter_conf_dir}":
-    command => "mkdir -p ${facter_conf_dir}",
-    unless  => "test -d ${facter_conf_dir}",
-    path    => '/bin:/usr/bin',
+  if $facts['os']['family'] == 'windows' {
+    exec { "mkdir_p-${facter_conf_dir}":
+      command => "cmd /c mkdir ${facter_conf_dir}",
+      creates => $facter_conf_dir,
+      path    => $::path,
```

Comment:
Is `$::path` a standard thing to be used in windows now?

Reply:
Yes, I think so

```
C:\ProgramData\PuppetLabs\puppet\etc>facter path
C:\Program Files\Puppet Labs\Puppet\puppet\bin;C:\Program Files\Puppet Labs\Puppet\facter\bin;C:\Program Files\Puppet Labs\Puppet\hiera\bin;C:\Program Files\Puppet Labs\Puppet\mcollective\bin;C:\Program Files\Puppet Labs\Puppet\bin;C:\Program Files\Puppet Labs\Puppet\sys\ruby\bin;C:\Program Files\Puppet Labs\Puppet\sys\tools\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files\Puppet Labs\Puppet\bin;C:\Users\tdockendorf\AppData\Local\Microsoft\WindowsApps
```